### ISSUE TITLE
wget do nothing

### ISSUE THREAD

**Issue by ale5000-git** [rmyorston/busybox-w32#75]
Why this doesn't download the file?
`busybox wget https://pypi.python.org/packages/source/s/six/six-1.0b1.tar.gz`

**Comment by rmyorston** [comment]
```busybox-w32``` doesn't support https.

In upstream BusyBox https is implemented using a helper application to handle the encryption but this requires a ```fork```, which isn't possible on Windows.

**Comment by ale5000-git** [comment]
I'm not so expert but isn't "ash" doing a fork also?

**Comment by rmyorston** [comment]
Yes.  There are a few places where busybox-w32 implements the equivalent of a fork.  Another instance is to allow tar to compress/decompress compressed archives.

However, there isn't a general solution to replacing fork.  Each case has to be implemented separately.

Support for https in wget is complex because the existing code uses both ```fork``` and ```socketpair```.  Given that https is becoming increasingly widely used, though, it's probably something I should look into.

Let's leave this issue open to serve as a reminder...

**Comment by ale5000-git** [comment]
I can think on three possible alternative ways:
- I know it wouldn't be the same but isn't possible to change the code to just use separate threads (like a normal multithreading application) rather than separate processes?
- Open a separate process and then doing interprocess communications: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365574(v=vs.85).aspx
- Implement a fork similar to Cygwin fork (quote: Currently, the Cygwin fork is a non-copy-on-write implementation similar to what was present in early flavors of UNIX)
